### PR TITLE
Add 00-intro.md

### PR DESCRIPTION
**Pull request by WinsonLiGeneration**
More comment can be typed here for more lines.
Such as line1
Line2
@winsonliwh

### CODE REVIEW

**Comment by winsonliwh** [comment]
It is good work!

File: _slides/00-intro.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Just add something new.
```
Comment (winsonliwh) [inline _slides/00-intro.md:1]: Good idea for adding something new.

Comment (winsonliwh) [reply _slides/00-intro.md:1]: After checking again, it really good!

**Review by winsonliwh — COMMENTED** [review]
I just submit general feedback without explicit approval of this.

**Review by winsonliwh — APPROVED** [review]
Approve

**Comment by WinsonLiGeneration** [comment]
@winsonliwh 

May I request your approval?

**Pull request merged**